### DISCUSSION TITLE
I'm facing this issue due to the donation process

### DISCUSSION THREAD

**Discussion by abysalvinthomas** [apohl79/audiogridder#264] [Q&A] [answered]
![Screenshot 2020-12-15 at 2 59 39 PM](https://user-images.githubusercontent.com/74089164/102196852-71592580-3ee6-11eb-8218-6013c3f2c878.png)

**Comment by abysalvinthomas** [comment]
is there any other way to do it?

**Comment by CaBuffalo** [comment]
Big thumb up. Let's keep this project going.

See what Andreas say... ...

**Comment by apohl79** [comment]
wow... didn't see that one coming! :-) let me see if i can find alternatives to paypal!

**Comment by apohl79** [comment, marked as answer]
Alternative: https://donorbox.org/audiogridder
